lib/cli/query.js

```diff
@@ -300,6 +300,9 @@ function argvQueryPrefold(argv, implicitTimestampOps) {
   if (argv.table === 'objects') {
     if (!query.filter[0][ts_attr] || query.filter[0][ts_attr].length === 0 &&
       implicitTimestampOps)
+      if (!query.filter[0][ts_attr]) {
```

Comment:
The `if` clause on line 301 that captured line 306 now does not capture the line. Is this intended?